### PR TITLE
Introduce a lobby + fix multiuser joining + instruction for deploying using a tunnelling system

### LINKED ISSUE #101: Support multiple human connecting to multiple trials

### PR DESCRIPTION
**Pull request by cloderic**
closes #101

### CODE REVIEW

File: cogment_verse/app.py [hunk 1]
```diff
@@ -83,10 +83,10 @@ def __init__(self, cfg, work_dir=".cogment_verse"):
             if service_type not in [service_type.value for service_type in ServiceType]:
                 raise RuntimeError(f"Unknown service type [{service_type}]")
             if service_type == ServiceType.ORCHESTRATOR.value:
```
Comment (wduguay-air) [inline cogment_verse/app.py:85]: Before it was considered a possibility to have more than one orchestrator?

**Review by wduguay-air — APPROVED** [review]
The multiplayer implementation in jsx is a bit over my head.

File: cogment_verse/app.py (hunk 1 above)
Comment (cloderic) [reply cogment_verse/app.py:85]: Yes (and it works fine) for the sake of simplicity I removed that for now (basically the idea is that if you have more than one orchestrator you'll have an "external" Cogment runtime (which is not yet supported).

File: cogment_verse/web/web_app/src/index.tsx [hunk 2]
```diff
@@ -15,12 +15,29 @@
...
+      <Route path="play" errorElement={<Error />}>
+        <Route index loader={playAnyLoader} />
+        <Route path=":trialId" element={<Play />} />
+      </Route>
+    </Route>
+  )
+);
 
 ReactDOM.render(
   <React.StrictMode>
```
[the remark this answers is not in the document]
Comment (cloderic) [reply cogment_verse/web/web_app/src/index.tsx:39]: Nope, it's been removed.

File: cogment_verse/processes/web.py [hunk 3]
```diff
@@ -57,12 +57,12 @@ def __init__(self, name, specs_filename, services_directory, web_cfg):
         served_dir = os.path.abspath(os.path.join(WEB_DIR, "build"))
```
[the remark this answers is not in the document]
Comment (cloderic) [reply cogment_verse/processes/web.py]: Right, I fixed it

**Review by wduguay-air — APPROVED** [review]

**Pull request merged**